go-controller/pkg/util/multi_network.go

```diff
@@ -530,6 +530,10 @@ func IsMultiNetworkPoliciesSupportEnabled() bool {
 	return config.OVNKubernetesFeature.EnableMultiNetwork && config.OVNKubernetesFeature.EnableMultiNetworkPolicy
 }
 
+func IsNetworkSegmentationSupportEnabled() bool {
+	return config.OVNKubernetesFeature.EnableMultiNetwork && config.OVNKubernetesFeature.EnableNetworkSegmentation
```

Comment:
We should return false if we are not at IC, since this feature is only support for it.

Reply:
@trozet wdyt? shall we support this feature for only nonIC?